src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -165,19 +170,17 @@ function MdAutocomplete ($mdTheming, $mdUtil) {
         </aria-status>';
 
       function getItemTemplate() {
-        var templateTag = element.find('md-item-template').remove(),
+        var templateTag = element.find('md-item-template').detach(),
             html = templateTag.length ? templateTag.html() : element.html();
         if (!templateTag.length) element.empty();
         return html;
       }
 
       function getNoItemsTemplate() {
-        var templateTag = element.find('md-not-found').remove(),
+        var templateTag = element.find('md-not-found').detach(),
```

Comment:
I believe this is just being used as a template (and angular isn't actually parsing it) - does it matter if we use remove or detach?